app.json

```
@@ -13,22 +13,22 @@
       "backgroundColor": "#ffffff"
     },
     "ios": {
-      "supportsTablet": true
+      "supportsTablet": true,
+      "bundleIdentifier": "com.bobbyxlr.my-app"
```

question: why have deviated from the set bundle names?

```suggestion
      "bundleIdentifier": "com.bobbyxlr.myPrismaApp"
```